### PR TITLE
fix: The organisation setting page is broken locally

### PR DESCRIPTION
**Pull request by novakzaballa**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [x] I have added information to `docs/` if required so people know about the feature!
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes

- Added the `plans` key in the project_local file

## How did you test this code?

- Locally go to the Organisation setting page in the main branch (See the error)
- Locally go to the Organisation setting page in this branch (No error)

### CODE REVIEW

**Review by matthewelwell — COMMENTED** [review]
These don't look right to me based on our conversation @kyle-ssg ?

**Review by khvn26 — APPROVED** [review]

**Comment by novakzaballa** [comment]
> These don't look right to me based on our conversation @kyle-ssg ?

@matthewelwell , I'm sorry, maybe I'm missing something, but this is not related to the announcement flag. It's related to changes in the Payment file, for example:

data-cb-plan-id={Project.plans.startup.annual}
Screenshot 2024-07-12 at 10 36 20 AM

File: frontend/env/project_local.js [hunk 1]
```diff
@@ -16,6 +16,10 @@ module.exports = global.Project = {
   flagsmithClientEdgeAPI: 'https://edge.api.flagsmith.com/api/v1/',
   // This is used for Sentry tracking
   maintenance: false,
+  plans: {
```
Comment (kyle-ssg) [inline frontend/env/project_local.js:19]:
```
  plans: {
    scaleUp: { annual: 'scale-up-annual-v2', monthly: 'scale-up-v2' },
    startup: { annual: 'startup-annual-v2', monthly: 'startup-v2' },
  },
```

Comment (matthewelwell) [reply frontend/env/project_local.js:19]: @novakzaballa this is what I was getting at.

**Review by kyle-ssg — APPROVED** [review]

**Pull request merged**